project/make/test-integration-cli

```diff
@@ -16,7 +16,7 @@ bundle_test_integration_cli() {
 	# even and especially on test failures
 	didFail=
 	if ! {
-		source "$(dirname "$BASH_SOURCE")/.ensure-busybox"
+		source "$(dirname "$BASH_SOURCE")/.ensure-frozen-images"
 		source "$(dirname "$BASH_SOURCE")/.ensure-emptyfs"
```

Comment:
Definitely not a stupid question -- the answer is that it's pretty arbitrary.  Back when it was `scratch` and we were building `busybox` from source, it actually mattered, but now it doesn't really matter much at all.